### DISCUSSION TITLE
[FEATURE]: getShopReceipts add an includes for transaction/images

### DISCUSSION THREAD

**Discussion by tk5446** [etsy/open-api#702] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

In the Etsy API v2 call to get shop receipts we used to be able to append to the querystring the "includes" parameter in order to retrieve an image url associated to each transaction like so:

`var uri = Uri.parse('https://openapi.etsy.com:443/v2/shops/${shopName}/receipts?includes=Transactions/MainImage(url_75x75),Transactions`');`

This doesn't seem to work anymore in v3.  

### Describe the solution you'd like

I do see that the transaction payload contains "listing_image_id" in the getShopReceipts v3 API call, but it seems like we will need to do a separate api call to get each transaction image.  I'm hoping to not have to do that.  Is there a way to get the image url on the new receipts call with the "includes" parameter or by other means?

**Comment by etsyachristensen** [comment]
At this time there is no way to get the image as part of the receipt object. You will have to call to the `getListingImage` endpoint to get the image. While I can add this to our list of discussion topics, I don't think it will be acted on any time soon, if at all.
